### PR TITLE
Add new dropdown for Projects

### PR DESCRIPTION
**Pull request by haideriqbal**
Add new project dropdown to filter experiments according to projects. Project information is sent in JSON payload as an array.

### CODE REVIEW

File: __test__/ExperimentTable.test.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import React from 'react'
-import { shallow, mount } from 'enzyme'
-import { getRandomInt, TableCellDiv, data, tableHeader } from './TestUtils'
```
Comment (lingyun1010) [inline __test__/ExperimentTable.test.js]: Actually you don't have to remove the space here, as this is we used when we import the undefault stuff.

File: src/ExperimentTable.js [hunk 2]
```diff
@@ -103,33 +113,46 @@ class ExperimentTable extends React.Component {
   }
 
   render() {
-    const { searchQuery, searchedColumnIndex, selectedSearch, selectedKingdom, checkedRows } = this.state
-    const { orderedColumnIndex, ascendingOrder } = this.state
```
Comment (lingyun1010) [inline src/ExperimentTable.js:107]: Please keep the spaces.

File: src/ExperimentTable.js [hunk 3]
```diff
@@ -103,33 +113,46 @@ class ExperimentTable extends React.Component {
...
+    const selectedSearchFilteredExperiments = this.sort(aaData).filter(data => data &&
+      Object.keys(data).map(key => displayedFields.includes(key) ? data[key] : null)
+        .some(value => value && value.toString().toLowerCase().includes(selectedSearch)))
 
-      this.sort(aaData).filter(data => data &&
-        Object.keys(data).map(key => displayedFields.includes(key) ? data[key] : null)
-          .some(value => value && value.toString().toLowerCase().includes(selectedSearch))) :
-      this.filter(this.sort(aaData), tableHeader)
-        .filter(data => selectedKingdom ? data.kingdom === selectedKingdom : true)
+    const dataArray = selectedSearch.trim() ?
```
Comment (lingyun1010) [inline src/ExperimentTable.js:133]: Good development.

**Review by lingyun1010 — APPROVED** [review]
Nice work. Please check code style comments below.

File: __test__/ExperimentTable.test.js (hunk 1 above)
Comment (haideriqbal) [reply __test__/ExperimentTable.test.js]: Okay will remove it. Its the intelliJ formatting which messed it up. Will avoid doing it in future

File: src/ExperimentTable.js (hunk 2 above)
Comment (haideriqbal) [reply src/ExperimentTable.js:107]: Fixed it!

**Pull request merged**